### PR TITLE
Added translations for Acknowledgments.jsonc

### PR DESCRIPTION
**Pull request by w41k3rc0m**
Added translations for zh-Hans, zh-Hant, and zh-HK.

### CODE REVIEW

**Comment by treastrain** [comment]
Thanks for your pull request!
The branch we're working on now is `v2.0-develop`, so choose that instead of `main` as the merge destination. (I changed that this PR.)

File: Strings/Acknowledgments.jsonc [hunk 1]
```diff
@@ -3,33 +3,33 @@
...
 
 	"ライブラリ": {
 		"ja": "ライブラリ",
 		"en": "Libraries",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "库",
+		"zh-Hant": "函數庫",
+		"zh-HK": "函式庫"
```
Comment (treastrain) [inline Strings/Acknowledgments.jsonc:16]: By "Libraries" here, I want to refer to third party's open source libraries, is that correct?

**Comment by w41k3rc0m** [comment]
Sorry. I'll make sure to do that for future pull requests. Thanks.

File: Strings/Acknowledgments.jsonc (hunk 1 above)
Comment (w41k3rc0m) [reply Strings/Acknowledgments.jsonc:16]: Yes. That is correct.

Comment (treastrain) [reply Strings/Acknowledgments.jsonc:16]: OK! Thank you!

**Pull request merged**